vector-stores/spring-ai-chroma-store/src/main/java/org/springframework/ai/vectorstore/ChromaVectorStore.java

```diff
@@ -162,4 +187,12 @@ public void afterPropertiesSet() throws Exception {
 		this.collectionId = collection.id();
 	}
 
+	@Override
+	public Builder createObservationContextBuilder(String operationName) {
+		return VectorStoreObservationContext.builder(VectorStoreProvider.CHROMA_VECTOR_STORE.value(), operationName)
+			.withDimensions(this.embeddingModel.dimensions())
+			.withCollectionName(this.collectionName + ":" + this.collectionId)
+			.withFieldName(this.initializeSchema ? DISTANCE_FIELD_NAME : KeyValue.NONE_VALUE);
```

Comment:
I would suggest keeping `KeyValue` concepts outside the vector store implementation and instead set it to `null`, so that we let the conventions handle "null" values (users might provide their implementation of the Conventions and decide to handle "null" values differently than `KeyValue.NONE_VALUE `).